### ISSUE TITLE
Methods beforeEach, beforeAll of bun:test is not working

### ISSUE THREAD

**Issue by paveltretyakovru** [oven-sh/bun#1280]
### Version

0.1.13

### Platform

Linux WIN-KNU6MIAGFN4 5.15.57.1-microsoft-standard-WSL2 #1 SMP Wed Jul 27 02:20:31 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux 

### What steps will reproduce the bug?

1. Touch a file **index.test.ts** with next content:

```typescript
import { it, describe, beforeAll, expect } from 'bun:test';

describe("check someObjct", () => {
  let someObject: { use(): void };
  
  beforeAll(() => {
    someObject = { use: () => console.log('Test use function') };
  });

  it('should have "use" method', () => {
    expect(typeof someObject.use).toBe("function");
  });
});
```

2. Run bun test
```bash
bun wiptest
```

### How often does it reproduce? Is there a required condition?

Everytime

### What do you see instead?

Exception:
```
 undefined is not an object (evaluating 'someObject.use')
```

![image](https://user-images.githubusercontent.com/7935943/192209632-08794950-0018-4f81-b77a-eeaf06d5a639.png)

The issue have same behaviro for ```beforeEach()``` function

![image](https://user-images.githubusercontent.com/7935943/192212446-694aeacd-1498-4f87-8e85-cf00bfd4de1a.png)

**Comment by ethanburrell** [comment]
Running into this issue too, going to take a crack at it

**Comment by Electroid** [comment]
This was fixed in #1639
